package.json

```diff
@@ -35,7 +35,7 @@
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
-    "react-native": "^0.59.5",
+    "react-native": ">=0.59.5 <0.64.0",
     "react-native-windows": "^0.57.2"
```

Comment:
Actually, it's worth mentioning that this should be updated with the same expression as above.

Reply:
```suggestion
    "react-native-windows": ">=0.59.5 <0.64.0"
```